pytorch_lightning/trainer/trainer.py

```diff
@@ -1075,6 +1077,91 @@ def tune(
 
         return result
 
+    def compressor(
```

Comment:
Let's remove the compressor method, we don't plan to add any new methods to the trainer.

Reply:
@tchaton ， the reasons is the same as above.
In INC tool, the tuning process used trainer.fit, trainer.validate and so on. if we use trainer.fit to quantize the model with INC, there is conflict when we use the early stop callback hook.

Reply:
We don't plan to add a new method to the Trainer. Therefore, we would either have to use more hooks within the Callbacks or add new hooks.

Reply:
removed